.travis.yml

```diff
@@ -21,8 +22,6 @@ jobs:
   include:
     - rvm: ruby-head
       gemfile: gemfiles/Gemfile.activesupport-edge
-    - rvm: jruby-head
-      gemfile: gemfiles/Gemfile.activesupport-edge
```

Comment:
I removed this one which is not passing and talking a lot of time, not a lot of value IMO.